src/main/app/src/app/admin/parameter-edit/parameter-edit.component.ts

```diff
@@ -581,6 +581,9 @@ export class ParameterEditComponent
     this.parameters.userEventListenerParameters =
       this.userEventListenerParameters;
 
+    this.parameters.intakeMail = this.mailFormGroup.get("intakeMail").value;
```

Comment:
ah... so easy. :-)

Reply:
Well, I am used to just dump all form data and have an occasional sanitise; but here; pfff